pkg/deploy/strategy/recreate/recreate.go

```diff
@@ -110,9 +115,23 @@ func (s *RecreateDeploymentStrategy) DeployWithAcceptor(from *kapi.ReplicationCo
 		return fmt.Errorf("couldn't decode config from deployment %s: %v", to.Name, err)
 	}
 
+	timeoutSeconds := defaultRetryTimeout
+	deployerPod, err := s.podClient.Pods(to.Namespace).Get(deployutil.DeployerPodNameForDeployment(to.Name))
+	if err != nil {
+		// if we are not able to get the deployer pod, set the timeout to default and just log
+		// the error.
+		glog.Errorf("unable to get deployer pod for %s/%s: %v", to.Namespace, to.Name, err)
```

Comment:
We don't really care about this error. I would suggest something like
```go
if err == nil {
  // set timeout - startTime
}
```
and be done with it